client/src/components/Explorer/ExplorerPanel.test.js

```diff
@@ -72,38 +72,40 @@ describe('ExplorerPanel', () => {
 
   describe('onHeaderClick', () => {
     beforeEach(() => {
-      mockProps.popPage.mockReset();
+      mockProps.gotoPage.mockReset();
     });
 
-    it('calls popPage', () => {
+    it('calls gotoPage', () => {
       shallow((
-        <ExplorerPanel {...mockProps} path={[1, 2, 3]} />
+        <ExplorerPanel {...mockProps} depth={1} />
       )).find('ExplorerHeader').prop('onClick')({
         preventDefault() {},
         stopPropagation() {},
       });
 
-      expect(mockProps.popPage).toHaveBeenCalled();
+      expect(mockProps.gotoPage).toHaveBeenCalled();
     });
 
-    it('does not call popPage for first page', () => {
+    it('does not call gotoPage for first page', () => {
+      mockProps.page.parent = null;
+
       shallow((
-        <ExplorerPanel {...mockProps} path={[1]} />
+        <ExplorerPanel {...mockProps} depth={1} />
```

Comment:
`depth` is already set to 1 in mockProps, so what I think you need here is just to override the `page` prop so it has no `parent`.